Engine/AlgorithmManager.cs

```diff
@@ -562,6 +563,16 @@ public void Run(AlgorithmNodePacket job, IAlgorithm algorithm, IDataFeed feed, I
                 //After we've fired all other events in this second, fire the pricing events:
                 try
                 {
+
+                    // TODO: For backwards compatibility only. Remove in 2017
+                    // For compatibility with Forex Trade data, moving 
+                    if (timeSlice.Slice.QuoteBars.Count > 0)
+                    {
+                        foreach (var tradeBar in timeSlice.Slice.QuoteBars.Where(x => x.Key.ID.SecurityType == SecurityType.Forex))
```

Comment:
no need for change here, but we often access `Symbol.ID.SecurityType` - perhaps it makes sense to define a `Symbol.SecurityType` for brevity